homeassistant/components/tplink/entity.py

```diff
@@ -292,15 +292,17 @@ def _description_for_feature[_D: EntityDescription](
                 translation_key=translation_key,
                 name=entity_name,  # if undefined will use translation key
                 entity_category=category,
-                entity_registry_enabled_default=enabled_default,
+                # enabled_default can be overridden to False in the description
+                entity_registry_enabled_default=(not is_debug_feature)
```

Comment:
```suggestion
                entity_registry_enabled_default=not is_debug_feature
```

Reply:
I don't think these are needed

Reply:
Fixed in https://github.com/home-assistant/core/pull/120141